source/beamline/BioXAS/BioXASZebra.cpp

```diff
@@ -131,8 +146,12 @@ void BioXASZebra::onConnectedChanged()
 	foreach (AMControl *orBlock, orBlocks_)
 		connected &= orBlock->isConnected();
 
+	foreach (AMControl *outputControl, outputControls_)
+		connected &= outputControl->isConnected();
```

Comment:
Just a suggestion. Will it make the codes looks better if we use `AMControlSet` here? If we put those controls into different AMControlSet, then we can use the `isConnected()` method and the `connected()` signal of AMControlSet